src/analytics.ts

```diff
@@ -340,6 +340,7 @@ export const setUserProperty = (
             return Promise.resolve();
         },
         onWeb(ga) {
+            // @ts-ignore
```

Comment:
I don't know how to avoid this ignore:
https://www.typescriptlang.org/play?#code/MYewdgzgLgBA8gIQFIwLwwN4DMQgFwwCMANDAEYCGATgQEymUBeBAzAL4wUQyiRQDcAKEEATAKbAANtTEwAbtRgQC0KgEswAcyGCA9Lp5dZhQYiQBtCAF0d+wxFm1BarDAAU3DfGQBKTIJhvC2shNmE7YCMYFmdXNzMAOgALLjgAdzAABSoQAAcxKigATw8fPwwAoMsbQTDBIA

Reply:
in that example this works: `OBJ[s as keyof typeof OBJ];`

Reply:
good enough for this case :+1: